improver/nowcasting/optical_flow.py

```diff
@@ -836,16 +822,11 @@ def process(self, cube1, cube2, boxsize=30):
         x_coord = cube2.coord(axis="x")
         y_coord = cube2.coord(axis="y")
         t_coord = cube2.coord("time")
-
         ucube = iris.cube.Cube(
             ucomp, long_name="precipitation_advection_x_velocity",
-            units="m s-1", dim_coords_and_dims=[(y_coord, 0), (x_coord, 1)])
-        ucube.add_aux_coord(t_coord)
-        amend_attributes(ucube, self.attributes_dict)
-
-        vcube = iris.cube.Cube(
-            vcomp, long_name="precipitation_advection_y_velocity",
-            units="m s-1", dim_coords_and_dims=[(y_coord, 0), (x_coord, 1)])
-        vcube.add_aux_coord(t_coord)
-        amend_attributes(vcube, self.attributes_dict)
+            units="m s-1", dim_coords_and_dims=[(y_coord, 0), (x_coord, 1)],
+            aux_coords_and_dims=[(t_coord, None)])
```

Comment:
Just to check, what's the reason that the creation of this output cube doesn't make use of the `generate_mandatory_attributes` functionality or add the mandatory attributes in some way to this output cube? I assume that the output from the `OpticalFlow` plugin and therefore the optical flow CLI wouldn't have the mandatory attributes. Is this because the optical flow output isn't regarded as a new diagnostic, so the mandatory attributes aren't required?

Reply:
The optical flow advection velocities aren't verifiable and are only used internally to the IMPROVER code.  I've sort of treated them conceptually as similar to the EMOS coefficients in that way.  But am open to other views.